examples/templates/README.md

```diff
@@ -30,14 +54,22 @@ In order to create new code examples with these templates, each new example must
     "url": "https://github.com/handsontable/handsontable/issues"
   },
   "author": "Handsoncode <hello@handsontable.com>",
-  "license": "MIT",
+  "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "handsontable": "latest",
-    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks
+    "@handsontable/{{ WRAPPER NAME }}": "latest" // only for frameworks, e.g. "@handsontable/react": "latest"
   }
 }
 ```
 
-#### Don't touch
+In `package.json` are also scripts ***you should never modify or remove***, because those scripts are used by our CI/CD pipeline : `npm run start`, `npm run build` and `npm run test`.
```

Comment:
WDYT
In the `package.json` you can find scripts that **should not be modified or removed**, such as `npm run start` or `npm run build`. We use them internally to run the jobs in the CI/CD pipeline.